deploy/awslambda/deploy.sh

```diff
@@ -19,9 +19,7 @@ aws cloudformation package --template-file "${template_file}" --output-template-
 aws cloudformation deploy --template-file output-cms.cf.yaml --stack-name "${ENVIRONMENT_NAME}-deploy" --parameter-overrides "EnvironmentName=${ENVIRONMENT_NAME}" "DomainName=${DOMAIN_NAME}" "CertificateArn=${CERTIFICATE_ARN}" "CMSSiteBucketURL=${ENVIRONMENT_NAME}-CMSSiteBucketURL" "CMSPostgresAddress=${ENVIRONMENT_NAME}-CMSPostgresAddress" "CMSPostgresPort=${ENVIRONMENT_NAME}-CMSPostgresPort" "AuroraMasterSecretARN=${ENVIRONMENT_NAME}-AuroraMasterSecretARN" "AuroraAppSecretARN=${ENVIRONMENT_NAME}-AuroraAppSecretARN" "CMSBlobStoreBucketName=${ENVIRONMENT_NAME}-CMSBlobStoreBucketName" "CMSRecordsWriterQueueURL=${ENVIRONMENT_NAME}-CMSRecordsWriterQueueURL" "CMSRecordsWriterQueueARN=${ENVIRONMENT_NAME}-CMSRecordsWriterQueueARN" "CMSImagesWriterQueueURL=${ENVIRONMENT_NAME}-CMSImagesWriterQueueURL" "CMSImagesWriterQueueARN=${ENVIRONMENT_NAME}-CMSImagesWriterQueueARN" "CMSPublisherQueueURL=${ENVIRONMENT_NAME}-CMSPublisherQueueURL" "CMSPublisherQueueARN=${ENVIRONMENT_NAME}-CMSPublisherQueueARN" "ElasticsearchDomainARN=${ENVIRONMENT_NAME}-ElasticsearchDomainARN" "LambdaFunctionSecurityGroup=${ENVIRONMENT_NAME}-LambdaFunctionSecurityGroup" "PrivateSubnet1=${ENVIRONMENT_NAME}-PrivateSubnet1" "PrivateSubnet2=${ENVIRONMENT_NAME}-PrivateSubnet2" "ElasticsearchDomainEndpoint=${ENVIRONMENT_NAME}-ElasticsearchDomainEndpoint" --capabilities CAPABILITY_IAM CAPABILITY_NAMED_IAM
 
 # echo Uploading static site content...
-pushd "${scriptdir}/../../client/"
-# This should probably be done in the Makefile
-npm run build
-popd
 aws s3 sync "${scriptdir}/../../client/dist/" "s3://${ENVIRONMENT_NAME}-site/" --delete
+# where should the search client go?
+#aws s3 sync "${scriptdir}/../../search-client/dist/" "s3://${ENVIRONMENT_NAME}-search-site/" --delete
```

Comment:
Could we copy it somewhere as part of the deploy script?

Reply:
I don't see why not. If you did something like:
```
aws s3 sync "${scriptdir}/../../search-client/dist/" "s3://${ENVIRONMENT_NAME}-site/search/" --delete
```
Assuming that syntax works (I haven't tested putting the subdirectory in the destination URL), you'd have to make sure the path you use didn't conflict with anything in the regular client, but we should be able to serve both of them out of the same bucket.

You'd have to add another route/integration combo in the CF similar to [this](https://github.com/OurRootsOrg/cms-server/blob/master/deploy/awslambda/cms-aurora.cf.yaml#L103-L122). 

Do you want to take a stab at it, or should I?

Reply:
Thanks. I'll take a stab at it. I'm out today but will be back tomorrow.